base/workflow/R/run.write.configs.R

```diff
@@ -25,6 +25,17 @@
 run.write.configs <- function(settings, write = TRUE, ens.sample.method = "uniform", 
                               posterior.files = rep(NA, length(settings$pfts)), 
                               overwrite = TRUE) {
+  samples.file <- file.path(settings$outdir, "samples.Rdata")
+  if (file.exists(samples.file)) {
+    samples <- new.env()
+    load(samples.file, envir = samples)
+    trait.samples <- samples$trait.samples
+    ensemble.samples <- samples$ensemble.samples
+    sa.samples <- samples$sa.samples
+  } else {
+    PEcAn.logger::logger.error(samples.file, "not found, this file is required by the run.write.configs function")
+  }
+  
```

Comment:
I think you'll need to move this down to replace the existing `load(...)` on line 97 -- the call to `get.parameter.samples` immediately above that is what writes `samples.Rdata`, so we need to do the reading after that.